ros/src/tl_detector/tl_detector.py

```diff
@@ -11,11 +14,8 @@
 
 from light_classification.tl_classifier import TLClassifier
 
-# from darknet_ros_msgs.msg import BoundingBox
-# from darknet_ros_msgs.msg import BoundingBoxes
-
-
 STATE_COUNT_THRESHOLD = 3
+SAVING_IMAGES = True
```

Comment:
Let's keep this as a ROS param, so we needn't recompile whenever we need to change this value.